app/components/validation/card.js

```diff
@@ -0,0 +1,229 @@
+  }
+
+  @action
+  deleteValidation() {
+    this.validation.subject = this.validationSubject;
+
+    this.args.delete(this.validation);
+  }
+
+  updateType = restartableTask(async (validation) => {
+    const { type } = validation;
+
+    if (type) {
+      this.validationType = {
+        object: type.subject,
+        predicate: RDF('type'),
+      };
+      this.validation.type = this.validationType;
+    }
+
+    this.validation.grouping = this.getGroupingForValidation(type.subject);
+
+    this.defaultErrorMessage = this.getDefaultErrorMessage();
+
+    if (!this.validation.resultMessage) {
+      this.validation.resultMessage = {
+        object: new Literal(this.defaultErrorMessage),
+        predicate: SHACL('resultMessage'),
+      };
+    } else {
+      this.validation.resultMessage.object.value = this.defaultErrorMessage;
+      await this.updateValidation.perform(this.validation);
+    }
+
+    // for updating the data in the child components
+    await timeout(1);
+  });
+
+  updateValidation = restartableTask(async (validation) => {
+    validation.type = this.validationType;
+
+    for (const property of Object.keys(validation)) {
+      this.validation[property] = validation[property];
+    }
+
+    if (isValidationConfigValidForType(this.validation)) {
+      this.isCardInError = false;
+      this.args.update(this.validation);
+    } else {
+      if (!this.isCardInError) {
+        this.isCardInError = true;
+      }
+    }
+  });
+
+  setup = restartableTask(async () => {
+    this.metaStore = new ForkingStore();
+    this.metaStore.parse(this.args.metaTtl, this.metaGraph, 'text/turtle');
+    this.setValidationTypes();
+
+    if (this.validation.type) {
+      this.validationType = this.validation.type;
+      this.defaultErrorMessage = this.getDefaultErrorMessage();
+      this.isCardInError = false;
+    } else {
+      this.isCardInError = true;
+    }
+  });
+
+  setValidationTypes() {
+    this.validationTypes = [];
+
+    if (!this.fieldDisplayType) {
+      return;
+    }
+
+    const conceptOptions = this.metaStore
+      .match(
+        this.fieldDisplayType,
+        EXT('canHaveValidation'),
+        undefined,
+        this.metaGraph
+      )
+      .map((triple) => triple.object);
+
+    const conceptScheme = new Namespace(
+      'http://lblod.data.gift/concept-schemes/'
+    )('possibleValidations');
+
+    const allOptions = this.metaStore
+      .match(undefined, SKOS('inScheme'), conceptScheme, this.metaGraph)
+      .map((t) => {
+        const label = getPrefLabelOfNode(
+          t.subject,
+          this.metaStore,
+          this.metaGraph
+        );
+
+        return { subject: t.subject, label: label && label.value };
+      });
+
+    const optionsForFieldType = allOptions.filter((option) => {
+      return conceptOptions
+        .map((concept) => concept.value)
+        .includes(option.subject.value);
+    });
+
+    const itemsWithDisabledOptions = optionsForFieldType.map((option) => {
```

Comment:
Adding a new empty one when a duplicate is found does not feel like the way to go tbh. if you have many duplicates it will add all empty validations. 

Achraf said that we indeed could just delete the duplicate validations but  that's not making the user aware of that this happend. When a bug is reported because validations are missing  I don't want to be the one that needs to find the if statement that checks if duplicate and than forcefully remove it.

We'll check with @aatauil

Reply:
Achraf said that it is good for now. Its _acceptable_